README.md

```diff
@@ -47,8 +47,9 @@ $ npm install
 ### Running on localhost:
 * **Step 1:** Fork susper repository and clone it to your desktop
 * **Step 2:** Then cd into that cloned folder
-* **Step 3:** Deploy locally by running this :```$ ng serve```
-* **Step 4:** Go to localhost:4200 where the application will be running locally.
+* **Step 3:** Then  hit ```$ npm install``` to install the node modules and the dependancies
```

Comment:
You have already made a PR (#919) with this change.

I don't think it is required here as well.

Reply:
Yeah it is not neede

Reply:
Although the picture changed previously with respect to the aspect ratios it doesn't fit correctly to its container we can see different parts of the image is missing in  different devices